Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.20-alpine
+
+WORKDIR /app
+RUN apk add --no-cache git
+RUN git clone https://github.com/lightninglabs/lightning-faucet.git .
```

Comment:
Not necessarily blocking, but we should likely start to tag versions of the project. I can push one out at the current commit once this is merged.